src/index/IndexImpl.h

```diff
@@ -158,6 +158,11 @@ class IndexImpl {
   NumNormalAndInternal numTriples_;
   string indexId_;
 
+  // Keeps track of the number of nonLiteral contexts in the index this is used
+  // in the test retrieval of the texts. This only works reliably if the
+  // wordsFile.tsv starts with contextId 1 and is continuous.
+  size_t nofNonLiterals_;
```

Comment:
The `IndexImpl` class contains the "normal" RDF index as well as the special text index. So this name (as well as the key with which it is stored in the configuration JSON whould reflect this.
So something like `numNonLiteralsInTextIndex` and `num-non-literals-text-index` would be my suggestion.